locales/en/actions.json

```diff
@@ -9,75 +9,54 @@
   "action-impact": "Impact",
   "action-implementation-phase": "Implementation phase",
   "action-last-updated": "Information updated",
-  "action-merged": "Merged actions",
-  "action-merged_CASE_STUDY": "Merged case studies",
-  "action-merged_STRATEGY": "Merged strategies",
+  "action-merged": "{context, select, CASE_STUDY {Merged case studies} STRATEGY {Merged strategies} other {Merged actions}}",
```

Comment:
@lilia1891 @terotik This is the main change to our translation files, translations based on context now follow this format under a single key ([docs](https://next-intl-docs.vercel.app/docs/usage/messages)).

Reply:
:white_check_mark: